Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -34,7 +34,7 @@ file that was distributed with this source code.
                 placeholder: '{{ placeholder ?: allowClearPlaceholder }}', // allowClear needs placeholder to work properly
                 allowClear: {{ required ? 'false' : 'true' }},
                 enable: {{ disabled ? 'false' : 'true' }},
-                readonly: {{ read_only is defined and read_only or attr.readonly is defined and attr.readonly ? 'true' : 'false' }},
```

Comment:
Same here, just a comment is enough.

`read_only` variable is already tested against definition on the template, I don't see how this could be buggy.